change/@internal-react-composites-5523c033-6f94-4a66-9fb6-5910d071fb8c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
nit: going by the comment `"changed meeting peopel pane to use ParticipantContainer Component."` this should be type: none as this is just an internal refactor (so no implication on customers).